backend/src/components/sections/lead-form.json

```diff
@@ -24,6 +24,9 @@
     },
     "description": {
       "type": "text"
+    },
```

Comment:
Although this add the message box in the component.  When making a form submission it will not the submitted message there.  

In order to post both email and message the change needs to be made in the [schema file](https://github.com/strapi/nextjs-corporate-starter/blob/main/backend/src/api/lead-form-submission/content-types/lead-form-submission/schema.json)